rails_docker.rb

```diff
@@ -104,6 +104,15 @@ def source_paths
   EOT
 end
 
+# Add @nimbl3/eslint-config-nimbl3 plugin to package.json
+insert_into_file 'package.json', before: %r{"version": .+\n} do
+  <<~EOT
+      "devDependencies": {
+        "@nimbl3/eslint-config-nimbl3": "^2.1.1"
```

Comment:
Use a fixed version `2.1.1`.

Actually it would be good to add a dot file `.npmrc` with the content:

```
save-exact=true
```

This ensures that only fixed versions are added.